R/repo-helper.R

```diff
@@ -0,0 +1,75 @@
+#' Create example repo for fledge demos
+#'
+#' @param open Whether to open the new project.
+#' @param pkg Package name.
+#' @param name Name for DESCRIPTION and git.
+#' @param email Email for DESCRIPTION and git.
+#' @param date String of time for DESCRIPTION and git.
+#' @param tempdir Directory within which to create the mock package folder.
+#' @param news If TRUE, create a NEWS.md file.
+#'
+#' @return
+#' @export
+#'
+create_fledge_mock_repo <- function(open = rlang::is_interactive(),
```

Comment:
Maybe we can `proj_activate()` if `open = FALSE`, and return the old active project invisibly?

```suggestion
create_demo_project <- function(open = rlang::is_interactive(),
```